docs/react-jss.md

```diff
@@ -229,25 +228,57 @@ const App = () => (
 )
 ```
 
-In case you need to access the theme but not render any CSS, you can also use `withTheme`. It is a Higher-order Component factory which takes a `React.Component` and maps the theme object from context to props. [Read more about `withTheme` in `theming`'s documentation.](https://github.com/iamstarkov/theming#withthemecomponent)
+#### Accessing the theme inside the styled component
+
+Pass the `injectTheme` option to `withStyles` so your theme will be injected into your wrapped component.
```

Comment:
Arn't we injecting the theme anyways if styles is a function?

Reply:
No only if `injectTheme` is true

Reply:
Oh I got confused by the example, it shows the theme being injected in to styles function but doesn't show it in the react component which made me assume we need injectTheme to pass it to the styles function. 

theme is getting passed to the styles function always! If no that would be evtl a problem.
theme is not getting passed with props to the user component by default, for this one needs injectTheme option, right? we need to make that difference clear in the docs if technically we are on the same page.